.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'enhancement'
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is.
+
+**Describe the solution you'd like**
+A clear and detailed description of what you want to happen.
```

Comment:
`A clear and detailed description of what you want to happen. Imagine you something new(Check, property, new behavior) that you requesting already implemented and you just report bug that feature does not work. Please report it in bug template format - <link>. `

Reply:
Changed